### ISSUE TITLE
Extract script throws error

### ISSUE THREAD

**Issue by evandavis** [APSL/redux-i18n#3]
After updating package.json and running `npm run extract`, I get the following error:

```shell
> i18n_extract

module.js:327
    throw err;
    ^

Error: Cannot find module 'glob'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/edavis/Projects/turo-web-client/node_modules/redux-i18n/bin/i18n_extract.js:2:12)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Function.Module.runMain (module.js:441:10)
```

Possibly a missing dependency?

**Comment by evandavis** [comment]
Looks like `glob` and `optimist` need to be added to `package.json`.

**Comment by francescarpi** [comment]
Thanks Evandavis!

I'm going to fix now.

**Comment by francescarpi** [comment]
Hi!

Fixed at tag 1.0.4. Can you confirm me thats ok, now?

Thank you very much.
Cesc

**Comment by evandavis** [comment]
Works as expected!
